src/lib/crypto/kimchi_backend/common/plonk_constraint_system.ml

```diff
@@ -436,13 +439,20 @@ struct
   (** Returns the number of public inputs. *)
   let get_primary_input_size t = Set_once.get_exn t.public_input_size [%here]
 
+  (** Returns the number of previous challenges. *)
+  let get_prev_challenges t = Set_once.get_exn t.prev_challenges [%here]
```

Comment:
I'm not sure I understand the utility of these functions, especially as they lose the `_exn`